logging/logadmin/logadmin_test.go

```diff
@@ -261,40 +262,59 @@ func TestFromLogEntry(t *testing.T) {
 	}
 }
 
+func TestListLogEntriesRequestDefaults(t *testing.T) {
+	const timeFilterPrefix = "timestamp >= "
+
+	got := listLogEntriesRequest("projects/PROJECT_ID", nil)
+
+	// parse time from filter
+	if len(got.Filter) < len(timeFilterPrefix) {
+		t.Errorf("expect default filter to start with %v; got %v", timeFilterPrefix, got.Filter)
+	}
+	filterTime, e := time.Parse(time.RFC3339, strings.Trim(got.Filter[len(timeFilterPrefix):], "\""))
+	if e != nil {
+		t.Errorf("expect default filter to have time in RFC3339; got %v", got.Filter)
+	}
+	timeDiff := time.Now().UTC().Sub(filterTime)
+
+	// Default is client's project ID, 24 hour lookback, and no orderBy.
+	if !testutil.Equal(got.ResourceNames, []string{"projects/PROJECT_ID"}) || got.OrderBy != "" || timeDiff.Hours() < 24 {
+		t.Errorf("expect {\"projects/PROJECT_ID\"} resource names, 24 hour look back filter and native order\ngot %v", got)
+	}
+}
+
 func TestListLogEntriesRequest(t *testing.T) {
-	dayAgo := time.Now().Add(-24 * time.Hour).UTC().Format(time.RFC3339)
 	for _, test := range []struct {
 		opts          []EntriesOption
 		resourceNames []string
 		filter        string
+		filterPrefix  string
```

Comment:
I think you can remove filter here, right? An exact filter prefix is just the filter.